script/functions.sh

```diff
@@ -59,6 +59,46 @@ function rspec_support_compatible {
   fi
 }
 
+function rspec_version_defined {
+  if [ "$RSPEC" != "" ]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+function run_spec_suite_for {
+  if [ ! -f ../$1/$SPECS_HAVE_RUN_FILE ]; then # don't rerun specs that have already run
+    if [ -d ../$1 ]; then
+      echo "Running specs for $1"
+      pushd ../$1
+      unset BUNDLE_GEMFILE
+      bundle_install_flags=`cat .travis.yml | grep bundler_args | tr -d '"' | grep -o " .*"`
+      travis_retry eval "bundle install $bundle_install_flags"
+      run_specs_and_record_done
+      popd
+    else
+      echo ""
+      echo "WARNING: The ../$1 directory does not exist. Usually the"
+      echo "travis build cds into that directory and run the specs to"
+      echo "ensure the specs still pass with your latest changes, but"
+      echo "we are going to skip that step."
+      echo ""
+    fi;
+  fi;
+}
+
+function run_other_repo_specs {
+  fold "rspec-core specs" run_spec_suite_for "rspec-core"
+  fold "rspec-expectations specs" run_spec_suite_for "rspec-expectations"
+  fold "rspec-mocks specs" run_spec_suite_for "rspec-mocks"
+  fold "rspec-rails specs" run_spec_suite_for "rspec-rails"
+
+  if rspec_support_compatible; then
+    fold "rspec-support specs" run_spec_suite_for "rspec-support"
+  fi
+}
```

Comment:
I don't think we need to run the specs for all the rspec repos as part of builds for this repo.